docassemble/ALToolbox/display_template.py

```diff
@@ -0,0 +1,35 @@
+import re
+from .copy_button import *
+
+def display_template(template, scrollable=True, collapse=False, copy=False, class_name='bg-light'):   
+  # 1. Initialize    
+  if scrollable:
+    scroll_class = 'scrollable-panel'
+    adjust_height = ''
+  else:
+    scroll_class = ''
+    adjust_height = f'onmouseover="this.style.height = (this.scrollHeight) + \'px\';"'       
```

Comment:
Do we know if this works on mobile devices? It seems unlikely to. Sorry if I'm going over old ground.

Reply:
I don't think so. The original copy button function uses similar mouse actions therefore not for mobile either. We would need a separate issue for that.

Reply:
Before I forget - where does this get changed back again? I can't find a `'onmouseout'`.

Reply:
I tested it on my phone and it works just fine! When I tap on the text, the textarea expands to its full height.

Reply:
It doesn't get changed back, because the idea is that it should be fully extended at the start, but it's not possible without JS. Good to hear that it works on mobile though.